labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -33,7 +33,40 @@ object CollectionExercise01 {
    *
    */
   def googleCodeJamGooglerese(lines: String*): Seq[String] = {
-    error("fix me")
+    // The solution answer to this is much cooler than this way;
+    // I consulted Kutsko and Petrack since I didn't feel like decoding
+    // the cypher.
+    lazy val mapping = Map(
+      'a' -> 'y',
+      'b' -> 'h',
+      'c' -> 'e',
+      'd' -> 's',
+      'e' -> 'o',
+      'f' -> 'c',
+      'g' -> 'v',
+      'h' -> 'x',
+      'i' -> 'd',
+      'j' -> 'u',
+      'k' -> 'i',
+      'l' -> 'g',
+      'm' -> 'l',
+      'n' -> 'b',
+      'o' -> 'k',
+      'p' -> 'r',
+      'q' -> 'z',
+      'r' -> 't',
+      's' -> 'n',
+      't' -> 'w',
+      'u' -> 'j',
+      'v' -> 'p',
+      'w' -> 'f',
+      'x' -> 'm',
+      'y' -> 'a',
+      'z' -> 'q',
+      ' ' -> ' '
```

Comment:
Haha, I did this the same way. I was hoping someone had thought of a better way to do it :P